Makefile

```diff
@@ -604,6 +605,10 @@ verify-modules: modules
 		echo "go module files are out of date"; exit 1; \
 	fi
 
+.PHONY: verify-conversions
+verify-conversions: $(CONVERSION_VERIFIER)  ## Verifies expected API conversion are in place
+	$(CONVERSION_VERIFIER)
```

Comment:
Could you add an echo as part of this target so that we can see its running?

Reply:
Done 👍